test/support/tests_for_national_applicability.rb

```diff
@@ -123,8 +111,49 @@ module TestsForNationalApplicability
       assert_equal detailed_guide.national_applicability, expected_national_applicability
     end
 
+    view_test "creating with all nations excluded should fail validation" do
+      create(:government)
+      attributes = attributes_for_edition(all_nation_applicability: "0")
+
+      all_nations = nation_inapplicabilities_attributes_for({
+        Nation.england => "http://www.england.com/",
+        Nation.scotland => "http://www.scotland.com/",
+        Nation.wales => "http://www.wales.com/",
+        Nation.northern_ireland => "http://www.ni.com/"
```

Comment:
Are these the official websites? If not, it would be better to use the standard url used for examples: `http://www.example.com/england`, `http://www.example.com/scotland`, etc.

Reply:
good idea